### PR TITLE
Made general project outline.

### PR DESCRIPTION
**Pull request by peterHoburg**
Docs are not completed. Some issues need to be created post merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
+directory.
+
+Find and replace the projects' name in these locations:
 
-#### Project Manager:
-- Joe Schmoe
+* Find all instances of `replaceme` and replace them with the projects' name.
+* Change the name of `./src/project_template` to `./src/<project_name>`
+* Change Github setting for the new repo
+    * Protect the main and staging branches with both actions.
```
Comment (Morpheus636) [inline README.md:15]: I think all of this stuff is done automatically since this is a template repo.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
+# Using this template
+
+The projects' name should be `lower_snake_case`. This should be the same name as the projects' root
+directory.
+
+Find and replace the projects' name in these locations:
 
-#### Project Manager:
-- Joe Schmoe
+* Find all instances of `replaceme` and replace them with the projects' name.
```
Comment (Morpheus636) [inline README.md:12]: Include adding a project description and usage instructions.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
-If you'd like to  join the core project team, which we're limiting to 10 people, you can reach out to a project manager or moderator and request to join. This will be first-come first-serve, and we will remove inactive contributors from the team  periodically to make space for new contributors.
+If it is a package then how to use/install
 
-If you can't get into the core team for whatever reason, or if you don't want to join the core team, you are still welcome to fork the project repository, and create a pull request to merge your changes with the rest of the project.
+# Documentation
 
-## Code Best Practices
-We're waiting on PeterHoburg for this.
+To view all the available documentation for this project please see
+our [Documents Index](/docs/INDEX.md). This includes how to get started, our coding best practices,
```
Comment (Morpheus636) [inline README.md:36]: Do we want this or GitHub Wiki?

File: docs/DOCUMENTATION.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# TODO
+HOW TO WRITE DOCS
```
Comment (Morpheus636) [inline docs/DOCUMENTATION.md:2]: The goal is to have someone else do this post-merge?

File: docs/GETTING_STARTED_DEVELOPING.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Follow these steps to get docker setup on your machine and get into shell inside the docker
+environment.
+
+* [Install docker on your machine](https://docs.docker.com/get-docker/)
+    * HINT: If on linux `curl -fsSL https://get.docker.com -o get-docker.sh; sudo sh get-docker.sh`
+* If on linux: [Install docker-compose on your machine](https://docs.docker.com/compose/install/)
+* Run `make initialize_pg`
+* Run `make interactive`
+
+You now have a local postgres database with fake data, and the production schemas linked to your
```
Comment (Morpheus636) [inline docs/GETTING_STARTED_DEVELOPING.md:44]: what is the purpose of fake data? Is this just temporary placeholder stuff that will be replaced in a real project?

File: docs/TEAM.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Team
+
+## Team lead
```
Comment (Morpheus636) [inline docs/TEAM.md:3]: Remove either team lead or project manager. It's redundant. 

Also add a spot for the rest of the team members.

**Review by Morpheus636 — COMMENTED** [review]
Looks mostly good. Just a couple of questions and suggestions in the comments in the review.

File: README.md (hunk 3 above)
Comment (peterHoburg) [reply README.md:36]: I looked at making a wiki, but you can't store the wiki docs in the main repo instead you have to pull a .wiki.git repo. It is annoying imo.

File: docs/GETTING_STARTED_DEVELOPING.md (hunk 5 above)
Comment (peterHoburg) [reply docs/GETTING_STARTED_DEVELOPING.md:44]: In a real project, only a couple of people should have access to the real data/production. The rest of the devs should have staging/beta/local envs with fake data to dev with.

File: docs/TEAM.md (hunk 6 above)
Comment (peterHoburg) [reply docs/TEAM.md:3]: Team lead and project managers are different roles in companies. The team lead would manage the devs and dev time, the project manager would manage the wider project.

File: README.md (hunk 3 above)
Comment (Morpheus636) [reply README.md:36]: Ok.

File: docs/GETTING_STARTED_DEVELOPING.md (hunk 5 above)
Comment (Morpheus636) [reply docs/GETTING_STARTED_DEVELOPING.md:44]: Oh ok. That makes sense.

File: docs/TEAM.md (hunk 6 above)
Comment (Morpheus636) [reply docs/TEAM.md:3]: So how would this work here? What would the team lead be doing vs the PM?

Comment (peterHoburg) [reply docs/TEAM.md:3]: PM's usually are not technical. They get requirements from clients, manage ticket priority, and other stuff like that. In this case, we don't need both. I would go with just a team lead, but they are technically different jobs.

Comment (Morpheus636) [reply docs/TEAM.md:3]: Ok Good to know. I'll update stuff elsewhere as well.

**Review by Morpheus636 — APPROVED** [review]
All good. Approved.

**Pull request merged**